### PR TITLE
Case search workflows

### PR DESCRIPTION
**Pull request by kiran-tripathy**
## Summary
Added Case Search Workflows

## Description

[Case Search Regression Plan](https://docs.google.com/spreadsheets/d/1sPLEAJFlhztZ656TH4jk-tjUbhu_YE-8dvVdgiy3xQQ/edit#gid=1311275056&range=F5)

### Link to Jira Ticket (if applicable)
[https://dimagi-dev.atlassian.net/browse/QA-4182](https://dimagi-dev.atlassian.net/browse/QA-4182)

## QA Checklist

- [x] Label(s) and Assignee added
- [x] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

**Review by kbo001 — CHANGES_REQUESTED** [review]

File: Features/CaseSearch/test_cases/conftest.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+@pytest.fixture(scope="session")
+def environment_settings_casesearch():
+    """Load settings from os.environ
+
+            Names of environment variables:
+                DIMAGIQA_URL
+                DIMAGIQA_LOGIN_USERNAME
+                DIMAGIQA_LOGIN_PASSWORD
+                DIMAGIQA_MAIL_USERNAME
+                DIMAGIQA_MAIL_PASSWORD
```
[the remark this answers is not in the document]
Comment (kiran-tripathy) [reply Features/CaseSearch/test_cases/conftest.py]: https://github.com/dimagi/dimagi-qa/pull/152/commits/7b88b6b0b4793c1cbf3ed350b11a772d49940367

File: Features/CaseSearch/test_cases/test_01_caseseach_workflows.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    """Checks if user can submit a form for skip to default search when re-searched """
+    webapps.open_app(case_search_app_name)
+    webapps.open_menu("Songs (Skip to Default Search)")
+    webapps.search_again_cases()
+    webapps.search_all_cases_on_case_search_page()
+    casesearch.check_default_rating_on_caselist("5")
+    case_name = webapps.omni_search("Bugs")
+    webapps.select_case(case_name)
+    webapps.open_form("Play Song")
+    webapps.submit_the_form()
```
[the remark this answers is not in the document]
Comment (kiran-tripathy) [reply Features/CaseSearch/test_cases/test_01_caseseach_workflows.py:129]: https://github.com/dimagi/dimagi-qa/pull/152/commits/40f3e454cf359736ad4fcaeaf2df216df5595f8f

File: Features/CaseSearch/test_cases/test_01_caseseach_workflows.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    """Checks if user can submit a form for normal search"""
+    webapps.login_as('automation-user-1')
+    webapps.open_app(case_search_app_name)
+    webapps.open_menu("Songs (Normal)")
+    webapps.search_all_cases()
+    webapps.search_all_cases_on_case_search_page()
+    case_name = webapps.omni_search("Bugs")
+    webapps.select_case(case_name)
+    webapps.open_form("Play Song")
+    webapps.submit_the_form()
```
[the remark this answers is not in the document]
Comment (kiran-tripathy) [reply Features/CaseSearch/test_cases/test_01_caseseach_workflows.py:20]: https://github.com/dimagi/dimagi-qa/pull/152/commits/40f3e454cf359736ad4fcaeaf2df216df5595f8f

File: Features/CaseSearch/test_cases/test_01_caseseach_workflows.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+
+def test_case_01_normal_workflow(driver):
+    webapps = WebApps(driver)
+    casesearch = CaseSearchWorkflows(driver)
+    """Checks if user can submit a form for normal search"""
+    webapps.login_as('automation-user-1')
+    webapps.open_app(case_search_app_name)
+    webapps.open_menu("Songs (Normal)")
+    webapps.search_all_cases()
```
[the remark this answers is not in the document]
Comment (kiran-tripathy) [reply Features/CaseSearch/test_cases/test_01_caseseach_workflows.py:15]: https://github.com/dimagi/dimagi-qa/pull/152/commits/40f3e454cf359736ad4fcaeaf2df216df5595f8f

**Review by kbo001 — APPROVED** [review]
approved

**Pull request merged**